### ISSUE TITLE
Shared Estate is not considered as a house

### ISSUE THREAD

**Issue by Lanicia** [NightmareXIV/Lifestream#66]
I am getting the error "[Lifestream] Could not find private house". (The house is registered in House Registration → Private House) So i think its because its shared estate and not directly my house.

**Comment by Limiana** [comment]
Yeah that makes sense, since shared estate is indeed a different thing than private house.
For shared estates, I'll add possibility to register their pathes in custom aliases in future update

**Comment by Lanicia** [comment]
Do you think it will be possible to have it in `/li auto` ?

**Comment by Limiana** [comment]
Hmm, I could add it there. The main issue comes when there's more than one shared estate, though

**Comment by meltryllis-git** [comment]
Went here to ask about this too! Please consider adding this even if it's limited to only the first shared estate!

**Comment by Limiana** [comment]
Shared estates have been added in testing version
